website2/.vuepress/local-components/SpeakerModal.vue

```diff
@@ -258,4 +272,24 @@ p {
 .text-center {
   text-align: center;
 }
+
+@keyframes modal-expand {
+  from {
+    width: 150px;
```

Comment:
Let’s animate with transform instead.

Reply:
I have implement with transform that make smooth animated.